mem0/vector_stores/azure_ai_search.py

```diff
@@ -60,13 +62,17 @@ def __init__(
                 Allowed values are None (no quantization), "scalar", or "binary".
             use_float16 (bool): Whether to store vectors in half precision (Edm.Half) or full precision (Edm.Single).
                 (Note: This flag is preserved from the initial implementation per feedback.)
+            use_hybrid_search (bool): Whether to use hybrid search. Default is False.
+            vector_filter_mode (Optional[str]): Mode for vector filtering. Default is "preFilter".
         """
         self.index_name = collection_name
         self.collection_name = collection_name
         self.embedding_model_dims = embedding_model_dims
         # If compression_type is None, treat it as "none".
-        self.compression_type = (compression_type or "none").lower()  
+        self.compression_type = (compression_type or "none").lower()
         self.use_float16 = use_float16
+        self.use_hybrid_search = use_hybrid_search
```

Comment:
```suggestion
        self.hybrid_search = hybrid_search
```

Lets call it `hybrid_search` only.